pyproject.toml

```diff
@@ -35,7 +35,7 @@ dependencies = [
   "matplotlib>=3.0.0,<4",
   "numpy>=1.10.0,<2",
   "pandas>1.0.0,<3",
-  "polars>=0.18.3,<0.19.0",
+  "polars>=0.18.3,<0.20.0",
```

Comment:
add here `polars>=0.19.0` instead of 0.18.3

Reply:
done